eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Wallet.hs

```diff
@@ -430,12 +427,10 @@ getRewardProvenance ::
   ) =>
   Globals ->
   NewEpochState era ->
-  (RewardUpdate (Crypto era), RewardProvenance (Crypto era))
+  (RewardUpdate (Crypto era))
 getRewardProvenance globals newepochstate =
```

Comment:
This is great @TimSheard , thanks for doing this clean up!  Before we merge this, we should make sure we coordinate with the consensus team about how best to deprecate their [query](https://github.com/input-output-hk/ouroboros-network/blob/a10ba425a5db0b7a009d6f16f066bb70b2bd3707/ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Query.hs#L250-L251). We've already gutted most of the information from that query, so it's really just about deprecating the query in the most clear and helpful way we can.